build.sbt

```diff
@@ -457,11 +457,14 @@ lazy val isScala3 = isScalaBinaryVersion("3")
 def isScala213or3 = Def.setting(isScala213.value || isScala3.value)
 
 lazy val munitLibrary = Def.setting {
+  // TODO: this is getting ridiculous; any way to fix dependency on munit's
+  // scala version upgrades, specifically for 2.13 (and not 2.12 or earlier)?
   val munitV =
     if (isScala211.value) "0.7.29"
     else if (scalaVersion.value == "2.13.11") "1.0.0-M10"
     else if (scalaVersion.value == "2.13.12") "1.0.0-M11"
     else if (scalaVersion.value == "2.13.13") "1.0.0"
+    else if (scalaVersion.value == "2.13.14") "1.0.2"
```

Comment:
any thoughts on why this is only necessary for 2.13 and not other minor versions?

Reply:
Scala 2.13.x will be able to break binary compat for the library by adding new methods etc. This will not happen for 2.12.x, since the work there will mostly be bugfixes and no new features

Reply:
so for each 2.13 patch we build, we will continue maintaining the appropriate version of munit?

and why only munit, and not any other dependency?